openmdao/docs/_utils/docutil.py

```diff
@@ -282,6 +329,16 @@ class or None
     indent = 0
     class_obj = None
     method_obj = None
+    
+    # check if it's a script excerpt
+    if len(path.split(':EXCERPT:')) > 1:
```

Comment:
I ultimately like this feature, and I think it would be a shame to limit it to just a single pair of tags in a function or file.  It should be workable even for tagged sections that are fully contained or cross boundaries with other tags.

With that said, I know that we are moving away from sphinx and towards notebooks (provided that all deficiencies can be solved), so maybe there isn't much justification for putting time into this...

Reply:
specificaly because we're moving away, I see no need to gold plate here. Limited functionality that can be grown later is fine with me.

Reply:
I had some discussions with outside folks who are using our docs tools about moving some of our sphinx embedding stuff into its own stand alone repo. So there will probably be life for this code after we move on anyway. I have no objection to including it in the codebase for now (assuming its apporpriately tested and passes code review)

Reply:
@Kenneth-T-Moore I'm pretty sure that overlapping or nested tags should work as written? I'll give it a try when I update the closing tag